src/Schema.js

```diff
@@ -244,9 +234,8 @@ class Schema {
         // createdAt and updatedAt are wacky and have legacy baggage
         parseFormatSchema.createdAt = { type: 'String' };
         parseFormatSchema.updatedAt = { type: 'String' };
-        this.data[schema.className] = _.mapValues(parseFormatSchema, parseField =>
-          schemaAPITypeToMongoFieldType(parseField).result
-        );
+        //Necessary because we still use the mongo type internally here :(
+        this.data[schema.className] = _.mapValues(parseFormatSchema, MongoSchemaCollection._DONOTUSEparseFieldTypeToMongoFieldType);
```

Comment:
As _colleciton is the MongoSchemaCollection, I propose that you do that transform in getAllSchemas() ? Later PR?